go.mod

```diff
@@ -100,29 +114,29 @@ require (
 	github.com/hashicorp/vault/api v1.0.5-0.20210210214158-405eced08457
 	github.com/hashicorp/vault/sdk v0.1.14-0.20210127185906-6b455835fa8c
 	github.com/influxdata/influxdb v0.0.0-20190411212539-d24b7ba8c4c4
-	github.com/jcmturner/gokrb5/v8 v8.0.0
+	github.com/jcmturner/gokrb5/v8 v8.4.2
 	github.com/jefferai/isbadcipher v0.0.0-20190226160619-51d2077c035f
 	github.com/jefferai/jsonx v1.0.0
 	github.com/joyent/triton-go v1.7.1-0.20200416154420-6801d15b779f
 	github.com/keybase/go-crypto v0.0.0-20190403132359-d65b6b94177f
 	github.com/kr/pretty v0.2.1
 	github.com/kr/text v0.2.0
 	github.com/lib/pq v1.8.0
-	github.com/mattn/go-colorable v0.1.7
+	github.com/mattn/go-colorable v0.1.8
 	github.com/mholt/archiver v3.1.1+incompatible
-	github.com/michaelklishin/rabbit-hole v0.0.0-20191008194146-93d9988f0cd5
-	github.com/mitchellh/cli v1.1.1
-	github.com/mitchellh/copystructure v1.0.0
+	github.com/michaelklishin/rabbit-hole v1.5.0
+	github.com/mitchellh/cli v1.1.0
+	github.com/mitchellh/copystructure v1.1.1
 	github.com/mitchellh/go-homedir v1.1.0
-	github.com/mitchellh/go-testing-interface v1.14.0
-	github.com/mitchellh/gox v1.0.1
-	github.com/mitchellh/mapstructure v1.3.3
+	github.com/mitchellh/go-testing-interface v1.14.2-0.20210217184823-a52172cd2f64
+	github.com/mitchellh/gox v0.4.0
```

Comment:
How come gox got downgraded?

Reply:
Good catch -- I think it had something to do with the versioning issue with go-testing-interface, but it was an unnecessary straggler dependency. Upgraded it back.